RestClientFile.rest

```diff
@@ -1,39 +1,80 @@
-
 ### Get All restaurants
 GET http://localhost:8080/restaurants
 
-
 ### Get All restaurants by filters
-GET http://localhost:8080/restaurants?name=Pahad poisid&location=Küüni 2, Tartu, Estonia
-
+GET http://localhost:8080/restaurants?name=Pahad%20poisid2&location=K%C3%BC%C3%BCni%202,%20Tartu,%20Estonia
 
 ### Get restaurant by id
 GET http://localhost:8080/restaurants/1
 
-
-### Add a restaurants
-POST  http://localhost:8080/restaurants HTTP/1.1
-content-type: application/json
+### Add a restaurant
+POST http://localhost:8080/restaurants
+Content-Type: application/json
 
 {
   "name": "Pahad poisid2",
   "location": "Küüni 2, Tartu, Estonia",
   "contactInfo": "pubi@pahadpoisid.ee"
 }
 
-
 ### Update a restaurant
-PUT   http://localhost:8080/restaurants/1 HTTP/1.1
-content-type: application/json
+PUT http://localhost:8080/restaurants/1
+Content-Type: application/json
 
 {
   "name": "Pahad poisid3",
   "location": "Küüni 2, Tartu, Estonia",
   "contactInfo": "pubi@pahadpoisid.ee"
 }
 
-
 ### Delete a restaurant
-DELETE    http://localhost:8080/restaurants/1
+DELETE http://localhost:8080/restaurants/1
+
+### Get Menu by Restaurant ID
+GET http://localhost:8080/restaurants/:id/menu
+
+### Create a Menu
+POST http://localhost:8080/restaurants/:id/menu
+Content-Type: application/json
+
+{
+  "lastUpdated": "2024-03-13T08:00:00"
+}
 
+### Delete Menu by ID
+DELETE http://localhost:8080/restaurants/:id/menu/:menuId
```

Comment:
Make sure these endpoints work in the local env

Reply:
I updated entities but still there is problem with rest api. So I pushed updates to branch for review if you can find the issue.

Reply:
Added small changes, so it should be working now and you can continue